### DISCUSSION TITLE
Can users use teleport without tsh?

### DISCUSSION THREAD

**Discussion by appetito** [gravitational/teleport#33974] [Q&A]
Hi!
Can users use teleport without tsh? Just  something like`ssh -i mykey username.targethost@proxy.com`?

**Comment by webvictim** [comment]
It's not possible for a user to log into a cluster without either running `tsh login` or using Teleport Connect (https://goteleport.com/docs/connect-your-client/teleport-connect/)


You can use `ssh` directly as a client once you've logged in with `tsh`, however.
